src/jit/emitfmtsxarch.h

```diff
@@ -111,7 +111,7 @@ IF_DEF(RRW_RRW_CNS, IS_R1_RW|IS_R2_RW,          SCNS)     // r/w    reg , r/w  r
 IF_DEF(RWR_RRD_RRD, IS_R1_WR|IS_R2_RD|IS_R3_RD, NONE)     // write  reg , read reg2 , read reg3
 IF_DEF(RWR_RRD_RRD_CNS, IS_R1_WR|IS_R2_RD|IS_R3_RD, SCNS) // write  reg , read reg2 , read reg3, const
 
-IF_DEF(RWR_RRD_RRD_RRD, IS_R1_WR|IS_R2_RD|IS_R3_RD|IS_R4_RD, NONE)     // write  reg , read reg2 , read reg3 , read reg4
```

Comment:
@fiigii, @CarolEidt. This was causing the additional assert, we need to ensure that `RWR_RRD_RRD_RRD` is marked as carrying a constant (for encoding `reg4`) so that the debug only `id` size checks succeed.

This seems to only matter when `reg4` is an extended register and generates "large" constant (which is really just a sign-extended byte constant)